### PR TITLE
Add python 3.7 for worker

### PR DESCRIPTION
**Pull request by pthegner**
related to BOT-1428

you can use the resulted image for development purpose (use on a docker-compose file).

Example : 

```
  persistor:
    image: par-vm232.srv.canaltp.fr:5000/kronos-worker:python3_7
    command: /bin/sh /var/www/Kronos/docker/launch_persistor.sh
    volumes:
      - ~/Kronos:/var/www/Kronos
    links:
      - database
      - rabbitmq
    env_file: ~/Kronos/docker/local.env
```

### CODE REVIEW

File: worker/build.sh [hunk 1]
```diff
@@ -4,6 +4,8 @@ OPTION=$1
 
 if  [[ $OPTION = "-debian8" ]]; then
     docker build -t par-vm232.srv.canaltp.fr:5000/kronos-worker:debian8 -f Dockerfile.debian8 .
+else if [[ $OPTION = "-python3_7" ]]; then
```
Comment (dvdn) [inline worker/build.sh]:
```suggestion
else if [[ $OPTION = "-python3" ]]; then
```

File: worker/Dockerfile.python3_7 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM python:3.7-alpine
+
+RUN apk add --no-cache postgresql-dev gcc musl-dev protobuf netcat-openbsd
```
[the remark this answers is not in the document]
Comment (pthegner) [reply worker/Dockerfile.python3_7]: https://askubuntu.com/questions/346869/what-are-the-differences-between-netcat-traditional-and-netcat-openbsd/426320#426320

File: worker/build.sh [hunk 3]
```diff
@@ -4,6 +4,8 @@ OPTION=$1
 
 if  [[ $OPTION = "-debian8" ]]; then
     docker build -t par-vm232.srv.canaltp.fr:5000/kronos-worker:debian8 -f Dockerfile.debian8 .
+else if [[ $OPTION = "-python3" ]]; then
```
Comment (Fahmus) [inline worker/build.sh]:
```suggestion
elif [[ $OPTION = "-python3" ]]; then
```

File: README.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ For debian7 (php5) images :
 For debian8 (php7) images :
 
     ./build.sh -debian8
+    
+For worker python 3.7 images :
+
+    ./worker/build.sh -python3
```
Comment (Fahmus) [inline README.md]:
```suggestion
    cd worker
    ./build.sh -python3
```

**Pull request merged**